tests/aws/services/events/helper_functions.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+from localstack.testing.aws.util import is_aws_cloud
+
+
+def is_v2_provider():
```

Comment:
This should go into some testing utility and not in the core codebase.

Reply:
Please ignore. Sorry, I missed the `tests` prefix. All good 👍 

We usually call it utils but helper_functions works as well 👍